src/protocols/rdp/keymap.h

```diff
@@ -103,6 +103,24 @@ struct guac_rdp_keymap {
      */
     const UINT32 freerdp_keyboard_layout;
 
+    /**
+     * FreeRDP keyboard type definition.
```

Comment:
If the keyboard layout, type, subtype, etc. are Windows/RDP standard values and not specific to FreeRDP, perhaps we should take this opportunity to move away from `freerdp_keyboard_layout`, `freerdp_keyboard_type`, etc. here (and `freerdp` within the keymaps) and migrate to naming that isn't tied to the underlying RDP implementation.

Reply:
It's good to migrate naming.
Should I replace these name in this PR?

Reply:
Yes, I would say so. With three new layout-related members being added here, if `freerdp_` is not an appropriate prefix, then it shouldn't be used for the new members, and the older, existing member should be corrected to follow the new convention. The member in question is not part of a public API, and avoiding the rename would amount to technical debt.